gitbutler-app/src/keys/controller.rs

```diff
@@ -14,7 +12,8 @@ impl Controller {
         Self { storage }
     }
 
-    pub fn from_path<P: AsRef<Path>>(path: P) -> Self {
+    #[cfg(test)]
+    pub fn from_path<P: AsRef<std::path::Path>>(path: P) -> Self {
```

Comment:
In this case it's because this method is only used in tests, hence `#[cfg(test)]`. If I did a `use std::path::Path` at the top, this would be the only usage of `Path`, and would cause a warning in normal (non-test) builds for being an unused `use`. Something I wish Rust handled better but alas, it doesn't.

In theory I could do a

```rust
#[cfg(test)]
use std::path::Path
```

but that's just adding more lines for no real gain. Thus, I just spell out the full canonical type to eliminate those two lines altogether.

Reply:
Awesome, makes sense!